src/app.js

```diff
@@ -132,6 +133,18 @@ var app = function(options) {
     });
   }
 
+  if (options.docs == undefined) {
+    options.rootDocsRedirect = false;
+  }
+
+  if (options.rootDocsRedirect) {
+    let link = options.docs.documenter.getDocumentationUrl();
+    DOCS_HTML = '<html><body><a href='+link+'>Refer to the documentation</a></body></html>';
+    app.get('/', function(req, res) {
+      res.send(DOCS_HTML);
```

Comment:
I believe we should set the response code to 404 here. You can use [res.status](http://expressjs.com/en/4x/api.html#res.status) to do so!